### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by evaveskova**
What was completed in this branch:

- Created a new react/redux app using "create-react-app"
- Deleted all unnecessary files
- Set up eslint and stickler
- Installed proptypes
- Deployed app to Heroku

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "prop-types": "^15.7.2",
```
Comment (bellom) [inline package.json:9]: image
You missed this 👆 
Kindly add it and re-submit for re-review.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 
### I hope you're not working alone on this project, if so, kindly explain the reason why.

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: package.json (hunk 1 above)
Comment (evaveskova) [reply package.json:9]: All necessary changes made. My partner and I are working together, this was just a very short milestone.

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**